roles/lnls-ans-role-repositories/molecule/default/tests/test_ubuntu_xenial_based.py

```diff
@@ -20,8 +20,7 @@ def test_repos(host, repo):
 
 # TODO: use epics version variables here?
 @pytest.mark.parametrize('pkg_pin', [
-    ('epics-dev', '3.15'),
-    ('epics-perl', 'none')
+    ('epics-dev', '3.15')
```

Comment:
We shouldn't change anything in the "repositories" role I think. It doesn't install anything. It just sets the "Pin" priority so we can install Debian packages in earlier versions of Ubuntu.

Reply:
ok, if you think epics-perl will ever be used, we can keep role-repositories as it is. done.